en/SUMMARY.md

```diff
@@ -609,6 +609,7 @@
         - [MountOrientation](msg_docs/MountOrientation.md)
         - [NavigatorMissionItem](msg_docs/NavigatorMissionItem.md)
         - [NavigatorStatus](msg_docs/NavigatorStatus.md)
+        - [NeuralControl](msg_docs/NeuralControl.md)
```

Comment:
Please remove this entry as per https://github.com/PX4/PX4-user_guide/pull/3617/files#r1980459242